### PR TITLE
Refactored description shows popups on every page

### PR DESCRIPTION
**Pull request by mynotdoing**
Refactored description shows popups on every page

https://github.com/ita-social-projects/SpaceToStudy-Client/assets/75339353/77920e86-28d8-4db7-a126-d57338cfce27

### CODE REVIEW

File: src/components/title-with-description/TitleWithDescription.tsx [hunk 1]
```diff
@@ -15,31 +14,37 @@ interface TitleWithDescriptionProps {
...
+      </Box>
+    )
+  } else {
+    return (
+      <Box sx={style.wrapper}>
+        <Typography sx={style.title}>{title}</Typography>
+        <Typography sx={style.description}>{description}</Typography>
+      </Box>
+    )
+  }
```
Comment (TSlashDreamy) [inline src/components/title-with-description/TitleWithDescription.tsx:47]: i think you can do it with less code, maybe something like this:
```js
    <Box sx={style.wrapper}>
      <Typography sx={style.title}>{title}</Typography>
      {useTooltip && (
        <Tooltip open={tooltipVisible} placement='bottom' title={description}>
          <Typography onClick={handleTooltip} sx={style.description}>
            {description}
          </Typography>
        </Tooltip>
      )}
    </Box>
  ```

Comment (mynotdoing) [reply src/components/title-with-description/TitleWithDescription.tsx]: fixed

File: src/components/title-with-description/TitleWithDescription.tsx [hunk 2]
```diff
@@ -15,12 +14,14 @@ interface TitleWithDescriptionProps {
...
   }
+  useTooltip?: boolean
 }
 
 const TitleWithDescription = ({
   title,
   description,
-  style = styles
+  style = styles,
+  useTooltip = false
```
Comment (Tolik170) [inline src/components/title-with-description/TitleWithDescription.tsx:24]: Maybe **withDescriptionTooltip** or **isDescriptionTooltip** name will be more clear

**Review by Tolik170 — APPROVED** [review]

**Review by OlyaKorchan — APPROVED** [review]

**Review by abalanovsky — APPROVED** [review]

File: src/components/title-with-description/TitleWithDescription.tsx (hunk 2 above)
Comment (mynotdoing) [reply src/components/title-with-description/TitleWithDescription.tsx]: fixed

**Pull request merged**